.github/workflows/build.yml

```diff
@@ -0,0 +1,51 @@
+name: Website build
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - id: nodeversion
+        run: echo ::set-output name=NODE::$(cat .nvmrc)
+      - name: restore Cache
+        uses: actions/cache@master
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+      - name: Use Node.js ${{steps.nodeversion.outputs.NODE}}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ steps.nodeversion.outputs.NODE }}
+      - name: install
+        run: |
+          yarn
+      - name: Lint
+        run: |
+          yarn run prettier:check
+          yarn run prettier:json:check
+          yarn run lint
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          start: yarn serve
+          wait-on: 'http://localhost:8080'
+      - name: build
+        run: |
+          echo VUE_APP_TRAVIS_COMMIT=$GITHUB_SHA >> .env.production
+          yarn run build
+          echo config.qmk.fm > dist/CNAME
+      - name: Deploy
```

Comment:
```suggestion
      - name: Deploy
        if: ${{ github.event_name != 'pull_request' }}
```

Reply:
This really depends on how you work regarding the difference between build and pr. doing the change